src/coreclr/tools/aot/crossgen2/Crossgen2RootCommand.cs

```diff
@@ -327,7 +327,7 @@ public static IEnumerable<Action<HelpContext>> GetExtendedHelp(HelpContext _)
                 Console.WriteLine();
 
                 string[] ValidArchitectures = new string[] {"arm", "armel", "arm64", "x86", "x64"};
-                string[] ValidOS = new string[] {"windows", "linux", "osx"};
+                string[] ValidOS = new string[] {"windows", "linux", "osx", "freebsd" };
```

Comment:
Nit: crossgen2 is also not necessary to build/use NativeAOT, so we could skip if this makes anything easier.